stream_chat/tests/async_chat/test_client.py

```diff
@@ -563,11 +574,52 @@ async def test_update_blocklist(self, client: StreamChatAsync):
     async def test_delete_blocklist(self, client: StreamChatAsync):
         await client.delete_blocklist("Foo")
 
+    async def test_check_push(
+        self, client: StreamChatAsync, channel: Channel, random_user: Dict
+    ):
+        msg = {"id": str(uuid.uuid4()), "text": "/giphy wave"}
+        await channel.send_message(msg, random_user["id"])
+        resp = await client.check_push(
+            {
+                "message_id": msg["id"],
+                "skip_devices": True,
+                "user_id": random_user["id"],
+            }
+        )
+
+        assert len(resp["rendered_message"]) > 0
+
     async def test_check_sqs(self, client: StreamChatAsync):
         response = await client.check_sqs("key", "secret", "https://foo.com/bar")
         assert response["status"] == "error"
         assert "invalid SQS url" in response["error"]
 
+    async def test_guest_user(self, client: StreamChatAsync, random_user: Dict):
+        try:
+            response = await client.set_guest_user({"user": {"id": str(uuid.uuid4())}})
+            assert "access_token" in response
+        except Exception:
+            # Guest user isn't turned on for every test app
+            pass
```

Comment:
this test is a bit useless like this, it will always pass

Reply:
it is, but I don't know if I want to mess with app settings in tests

Reply:
even if the request succeeds but the assert fails, it will still be ignore, no?

Reply:
that's actually a good point @gumuz 🤦 I'm making it catch `StreamException`

Reply:
Pushed. ✅